monai/transforms/croppad/array.py

```diff
@@ -838,8 +838,8 @@ def __init__(
                 of image. if None, select foreground on the whole image.
             margin: add margin value to spatial dims of the bounding box, if only 1 value provided, use it for all dims.
             allow_smaller: when computing box size with `margin`, whether allow the image size to be smaller
```

Comment:
thanks, now I understand the logic, maybe the docstring could be enhanced:
```suggestion
            allow_smaller: when computing box size with `margin`, whether to allow the final box edges to be outside of the image edges (the image is smaller than the box). If `True`, part of a padded output box might be outside of the original image, if `False`, the image edges will be used as the box edges.
```

Reply:
Hi Wenqi, seems the meaning of your enhanced docstring is opposite.
If True, means the margined size can be larger than the image size, so it will pad the output.
If False, means the margined size can not be larger than the image size, so it will not pad the output.
Correct me if I'm wrong.

Reply:
Yes I didn't see your change in the transform utils when commenting... The code logic looks good to me, perhaps the docstring still needs to clarify the current default behaviour when allow_smaller is False.

Reply:
Thanks! Have updated the docstring based on your comments!